quattor/client/rpms.pan

```diff
@@ -29,6 +28,11 @@ include { 'rpms/package_default_versions' };
     if(! exists(spma)) {
         SELF[spma] = nlist();
     };
+    ncd = escape('ncm-ncd');
```

Comment:
The logic---"add if not present but do not overwrite a version lock"---is the purpose of pkg_add() so you can just use that instead. Unless the comment on line 26 is suggesting that components/spma/config cannot be included here?